### ISSUE TITLE
Calculation of Monthly Report needs to include the "end date" inserted 

### ISSUE THREAD

**Issue by OritGeron** [Gizra/productivity#528]
![unfpa_po__23_-_12408__global__new___live_gizra_productivity](https://user-images.githubusercontent.com/7068965/35725622-ad587ace-080a-11e8-97e3-c41e5686ec57.jpg)

![unfpa_po__23_-_12408__global__new___live_gizra_productivity](https://user-images.githubusercontent.com/7068965/35725650-caa41156-080a-11e8-8c47-638712abdd5b.jpg)
![unfpa_po__23_-_12408__global__new___live_gizra_productivity](https://user-images.githubusercontent.com/7068965/35725684-ea9855c6-080a-11e8-88cc-7a3c3c863c70.jpg)

![unfpa_po__23_-_12408__global__new___live_gizra_productivity](https://user-images.githubusercontent.com/7068965/35725702-fa516b74-080a-11e8-86c4-3a8e49a12aca.jpg)

Bottom line - January 31st. is not included in the calculation if you insert it in the end date.

**Comment by bricel** [comment]
solved with this commit 6e89340911a0be610ce151e5d7452dd851ef5e0c
